docs/canvas/canvas-present-workpad.asciidoc

```diff
@@ -0,0 +1,30 @@
+[role="xpack"]
+[[canvas-present-workpad]]
+== Present your workpad
+
+When you are ready to present your workpad, use and enable the presentation options.
+
+[float]
+[[view-fullscreen-mode]]
+=== View your workpad in fullscreen mode
+
+In the upper left corner, click the *Enter fullscreen mode* icon.
+
+[float]
+[[zoom-in-out]]
```

Comment:
Zoom in and out isn't really a presentation feature. It might be better to stick this on the `canvas-elements` page. The zoom level you set while editing the workpad isn't applied in fullscreen mode. The page will just stretch to fit the entire window.